Podcast/PlayerControlsView.swift

```diff
@@ -143,6 +151,15 @@ class PlayerControlsView: UIView {
     func sliderValueChanged() {
         delegate?.playerControlsDidScrub()
     }
+    
+    func moreButtonTapped() {
+        delegate?.playerControlsDidTapMoreButton()
+    }
+    
+    func recommendButtonTapped() {
+        recommendButton.isSelected = !recommendButton.isSelected
+        delegate?.playerControlsDidTapRecommendButton()
```

Comment:
add backend call to the delegate that implements this protocol (aka the delegate playerControlsDidTapRecommendButton should have a endpoint request)